### PR TITLE
Add support for color hsl

### PR DESCRIPTION
**Pull request by qur786**

### CODE REVIEW

**Comment by qur786** [comment]
HSLColor

File: src/utils.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
+import { hsl } from 'd3';
 import { interpolateSpectral, scaleSequential, scaleLinear } from 'd3';
 
+const getHCL = (val) => hsl(360 * val, 0.85, 0.7);
```
Comment (maneetgoyal) [inline src/utils.js]:
```suggestion
function getHSL(val) ....
```

File: src/utils.js [hunk 2]
```diff
@@ -14,6 +17,11 @@ export function createColorGenerator(min_value, max_value, color) {
         .range(['#ffffff', '#ff4500'])
         .domain([min_value, max_value]);
       break;
+    case 'hsl':
```
Comment (maneetgoyal) [inline src/utils.js]: put it above null/undefined case

**Review by maneetgoyal — APPROVED** [review]
update storybook via yarn build before merging

**Pull request merged**